tests/go/GoTOMLBearTest.py

```diff
@@ -0,0 +1,34 @@
+from bears.go.GoTOMLBear import GoTOMLBear
+from coalib.testing.LocalBearTestHelper import verify_local_bear
+
+good_file = """package main
+import "fmt"
+func main() {
+    fmt.Println("Hello World!")
+}
+"""
```

Comment:
I don't understand, why are you using a GO file to check a TOMLBear, the input file for a TOML bear should be a TOML ....

I find tests completely irrelevant :thinking: ?

AFAIU @yukiisbored 'd proposal was of a bear that can lint `.toml` files, am I right @yukiisbored ?